tests/cypress/tests/server/select_row_test.ts

```diff
@@ -22,4 +23,56 @@ describe('select row', () => {
             DashTable.getSelect(0).within(() => cy.get('input').should('not.be.checked'));
         });
     });
+
+    describe('fe pagination & sort', () => {
```

Comment:
New tests for derived selected rows props -- check that the Dash Server receives the correct values for derived_virtual_selected_rows and derived_viewport_selected_rows with/without sort and filtering